db/seeds.rb

```diff
@@ -35,7 +35,7 @@ def build_menus_with_dishes(menu_names)
   Restaurant.all.each do |restaurant|
     menu_names.each do |menu_name|
       #menu = Menu.create(name: menu_name, restaurant: restaurant)
-      menu = Menu.create(name: menu_name)
+      menu = Menu.create(title: menu_name, restaurant: Restaurant.all.sample)
```

Comment:
The `Restaurant.all.sample` doesn't make sense since you are inside a loop and have `restaurant` available. Or am I missing something?